hypchat/restobject.py

```diff
@@ -156,7 +157,7 @@ def notification(self, message, color=None, notify=False, format=None):
                 format = 'text'
             else:
                 format = 'html'
-        data = {'message': message, 'notify': notify, 'message_format': format}
+        data = {'message': message, 'card': card, 'notify': notify, 'message_format': format, 'attach_to': attach_to}
```

Comment:
I tried this code, and if `attach_to` is None, there will be error like below:

```
hypchat.requests.HttpBadRequest: {
  "error": {
    "code": 400,
    "description": "Value {value!r} is not of type {expected_type!r}",
    "expected_type": "string",
    "field": "attach_to",
    "message": "Value None for field 'attach_to' is not of type 'string'",
    "type": "Bad Request",
    "validation": "type",
    "value": null
  }
```

Seems to use `None` check will be better:

```python
if card:
    data['card'] = card
if attach_to:
    data['attach_to'] = attach_to
```